source/administration/config-settings.rst

```diff
@@ -2345,6 +2360,18 @@ Id Attribute
 | This feature's ``config.json`` setting is ``"IdAttribute": ""`` with string input.                                                                                   |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Guest Attribute
+^^^^^^^^^^^^^^^^^
+(Optional) The attribute in the SAML Assertion used to apply a guest role to users in Mattermost.
+
+See the `Guest Accounts documentation <https://docs.mattermost.com/deployment/guest-accounts.html>`__ for more information
+(available in Enterprise Edition E20 and higher).
```

Comment:
Following the format of other config settings, this should be *Available in Enterprise Edition E20* (omit paretheses and remove "and higher"